test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -284,5 +284,54 @@ public void DnuPack_ShowUnresolvedDependencyWhenBuildFails(string flavor, string
                 Assert.Equal(1, unresolvedDependencyErrorCount);
             }
         }
+
+        [Theory]
+        [MemberData(nameof(RuntimeComponents))]
+        public void DnuPack_ResourcesNoArgs(string flavor, string os, string architecture)
+        {
+            string stdOut;
+            string stdError;
+            var runtimeHomeDir = TestUtils.GetRuntimeHomeDir(flavor, os, architecture);
+            int exitCode;
+            string netFlavorFolder = "dnx451";
+            if (flavor != "clr")
+            {
+                netFlavorFolder = "dnxcore50";
+            }
+            const string testApp = @"ResourcesTestProjects\ReadFromResources";
```

Comment:
Don't use backslash, this will cause issues when we run the test on non-windows machines